### PR TITLE
New typings for fs-cson module

### PR DESCRIPTION
**Pull request by dex4er**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by RyanCavanaugh** [comment]
@dex4er Please fix the failures indicated in the Travis CI log.

File: types/fs-cson/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// Type definitions for fs-cson 0.3
+// Project: https://github.com/charlierudolph/fs-cson
+// Definitions by: Piotr Roszatycki <https://github.com/dex4er>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+export function readFile(filePath: string, done: (err: Error | null, result: any) => void): void;
+export function readFileSync(filePath: string): any;
+export function register(): any;
```
Comment (andy-ms) [inline types/fs-cson/index.d.ts]: Seems like this returns `void`?

Comment (dex4er) [reply types/fs-cson/index.d.ts]: I'm not sure. `register()` is defined here  https://github.com/charlierudolph/fs-cson/blob/ea5420ec9001f6abd4f279d83ae2f38187100335/src/register.coffee#L5

and

```
> require('fs-cson').register()
{}
```

and it looks like returns what is returned by `readFileSync`. But I would agree that doesn't make any sense to expect any result from `register` function so I'm going to change it to `void`.

**Comment by andy-ms** [comment]
Thanks!

**Pull request merged**